src/store/modules/user.js

```diff
@@ -0,0 +1,63 @@
+import { userService } from "../../services"
+import router from "@/router/index.js"
+
+const user = JSON.parse(localStorage.getItem("user"))
+const state = user
+	? { status: { loggedIn: true }, user }
+	: { status: {}, user: null }
```

Comment:
it's not good to use optional check when you create a store entity try to avoid it your store object should be initially same